caesars-palace/styles/styles.css

```diff
@@ -30,7 +30,8 @@
   /* fonts */
   --body-font-family: 'Open Sans',sans-serif;
   --heading-font-family: 'Josefin Sans',sans-serif;
-   --fixed-font-family: "Roboto Mono", menlo, consolas, "Liberation Mono", monospace;
+  --fixed-font-family: "Roboto Mono", menlo, consolas, "Liberation Mono", monospace;
+  --montserrat-font-family: 'Montserrat',sans-serif;
```

Comment:
CSS var names should abstract away what they are defining. In this case calling the var `montserrat-font-family` isn't very useful. I think a better way would have been to override the `--heading-font-family` within your component that wanted to use a different font.